driver_cpl/cime_config/config_component_acme.xml

```diff
@@ -352,80 +498,114 @@
+      <value compset="^2000.+AV1C-04P"				>368.865</value>
+      <value compset="^2000.+AV1C-04P2"				>368.865</value>
+      <value compset="^2000.+AV1C-L"				>368.865</value>
       <!-- Override values based on CAM (WACCM) chemistry -->
-      <value compset="CAM[456]0%W"               >0.000001</value>
-      <value compset="CAM[456]0%SVBS"            >0.000001</value>
-      <value compset="CAM[456]0%TMOZ"            >0.000001</value>
+      <value compset="CAM[45]%W"				>0.000001</value>
+      <value compset="CAM[45]%SSOA"				>0.000001</value>
       <!-- Mariana's fix to allow isotopes to get the proper CO2 value -->
-      <value compset="CAM..%WISOall"             >284.7</value>
-      <value compset="^2000.+CAM[45]%WCSC"       >368.9</value>
-      <value compset="^2000.+CAM[45]%MOZM"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%MMOS"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%TMOZ"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%SMA3"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%SMA7"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%SSOA"       >0.000001</value>
+      <value compset="CAM[45]%WISOall"				>284.7</value>
+      <value compset="^2000.+CAM[45]%WCSC"			>368.9</value>
+      <value compset="^2000.+CAM[45]%MOZM"			>0.000001</value>
+      <value compset="^2000.+CAM[45]%MMOS"			>0.000001</value>
+      <value compset="^2000.+CAM[45]%TMOZ"			>0.000001</value>
+      <value compset="^2000.+CAM[45]%SMA3"			>0.000001</value>
+      <value compset="^2000.+CAM[45]%SMA7"			>0.000001</value>
+      <value compset="^2000.+CAM[45]%SSOA"			>0.000001</value>
     </values>
     <group>run_co2</group>
     <file>env_run.xml</file>
     <desc>This set the namelist values of CO2 ppmv for CAM and CLM. This variables is
       introduced to coordinate this value among multiple components.</desc>
   </entry>
 
-  <entry id="CO2_TYPE_COSTANT_VALUE">
-    <type>real</type>
-    <valid_values></valid_values>
-    <default_value>284.7</default_value>
-    <values>
-      <value compset="^2000">367.0</value>
-      <value compset="^4804">367.0</value>
-    </values>
-    <group>run_co2</group>
-    <file>env_run.xml</file>
-    <desc>
-      Mechanism for setting the CO2 value in ppmv for CLM if
-      CLM_CO2_TYPE is constant POP if OCN_CO2_TYPE is constant.
-    </desc>
+  <entry id="SCIENCE_SUPPORT">
+    <type>char</type>
+    <valid_values>on, off</valid_values>
+    <default_value>off</default_value>
+    <group>case_def</group>
+    <file>env_case.xml</file>
+    <desc>If set to off, this component set/ grid specification is not scientifically supported.
+    If set to on, this component set/ grid specification is scientifically supported</desc>
   </entry>
 
-  <entry id="FLDS_WISO">
-    <type>logical</type>
-    <valid_values>TRUE,FALSE</valid_values>
-    <default_value>FALSE</default_value>
+  <entry id="CCSM_CCOST">
```

Comment:
We still had this and CCSM_GCOST in the 5.1 branch we started with.  I'm not sure if they are used for anything and fine with removing them.